internal/server/kong/ws/server.go

```diff
@@ -85,6 +87,7 @@ func validateRequest(r *http.Request) error {
 }
 
 func (h websocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	dpConnectStartTime := time.Now()
```

Comment:
Should we move this right above `m, err := h.authenticator.Authenticate(r)`? As the way it currently is, it'd capture a bit more than just connecting.

Reply:
@mikefero wdyt? You mentioned we should include request validation when we talked about this metric. I don't have a strong opinion either way.

Reply:
The strange thing to me is the naming, as the metric is named to make it think it's the connection duration only. It's fine for including the request validation, but it seemed a little strange.

Either way, both are part of establishing a connection. Whenever we switch this over to the Prometheus metric format, we can at least provide a help message documenting what it's capturing.

Reply:
I am fine with having the name indicate this is a validated authenticated connection duration; more verbosity in the name doesn't hurt as we could add the other metrics during the connection phases later on.

Reply:
Ohh I'm fine with how it is right now, just that it's slightly confusing as mentioned previously. Technically speaking this is the time for authentication & the actual connection. Perhaps instead of saying "connect", "authenticate" may be a better word.